.github/workflows/backend-typing.yml

```
@@ -0,0 +1,81 @@
+name: backend typing
+on:
+  push:
+    branches:
+      - master
+      - releases/**
+  pull_request:
+
+jobs:
+  test:
+    name: backend typing
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
```

```suggestion
    timeout-minutes: 12
```

I see it taking 7 minutes, thus, reducing it.